crates/neon-sys/package.json

```diff
@@ -1,14 +1,10 @@
 {
   "main": "index.js",
   "scripts": {
-    "configure-release": "node-gyp configure",
-    "build-release": "node-gyp build",
-    "configure-debug": "node-gyp configure -d",
-    "build-debug": "node-gyp build -d"
+    "preinstall": "echo 'Skipping node-gyp installation as part of npm install.'"
   },
   "devDependencies": {
-    "nan": "^2.3.2",
-    "node-gyp": "^3.3.1"
```

Comment:
It's installed with node (technically, npm) so yes.  Depending on a specific node-gyp version is an anti-pattern in most cases.

I get bug reports from time to time where people complain node-gyp stopped working for them and then it turns out they've pinned it to a five years old release.